geoips/plugins/modules/algorithms/pmw_tb/pmw_89pct.py

```diff
@@ -10,16 +10,18 @@
 # # # for more details. If you did not receive the license, for more information see:
 # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-"""Data manipulation steps for "89pct" product.
+"""Passive Microwave 89 GHz Polarization Corrected Temperature.
 
+Data manipulation steps for the "89pct" product.
 This algorithm expects Brightness Temperatures in units of degrees Kelvin
 """
 import logging
 
 LOG = logging.getLogger(__name__)
 
+interface = "algorithms"
 family = "list_numpy_to_numpy"
-description = "Passive Microwave 89 MHz Polarization Corrected Temperature"
+name = "pmw_89pct"
```

Comment:
@jsolbrig I updated these from "pmw_tb.pmw_89pct" to just "pmw_89pct".

The only catch here is that now the algorithm pmw_89pct and colormap pmw_89pct have the same names - it will not conflict since they are in different interfaces, but will it cause unnecessary confusion?

Reply:
@mindyls I think this is good. I would like to come up with better names for the colormaps at some point, though, like we've discussed. We should talk with a MW expert to see if they can give names to the different frequency ranges. I imagine we could probably follow something like the common band names: https://www.everythingrf.com/tech-resources/frequency-bands